frontend/src/app/search-bar/search-bar.component.scss

```diff
@@ -1,9 +1,8 @@
 .mat-mdc-chip {
   position: relative;
-  background-color: white!important;
+  color: white!important;
```

Comment:
comment: Please avoid using `!important`. If it really is necessary, be sure to put a space after the style value and `important`. 

See:
https://google.github.io/styleguide/htmlcssguide.html#Important_Declarations

Reply:
This issue still needs to be addressed.